### DISCUSSION TITLE
Conda package

### DISCUSSION THREAD

**Discussion by kir0ul** [copier-org/copier#734] [Show and tell]
Hi all,

We mainly use Conda to install Python packages, and we had a Copier template for a while that we had some trouble making it working smoothly across platforms using Pip from a Conda environment (there were some issues in building Copier on Windows if I remember correctly), so to solve this I took the liberty to build Copier for Conda and push it to [Conda-forge](https://conda-forge.org/).

Here's the [package on Anaconda.org](https://anaconda.org/conda-forge/copier) and here's the [recipe I used for the build](https://github.com/conda-forge/copier-feedstock/blob/main/recipe/meta.yaml).

Let me know if you have any comment about the Conda recipe or if you want to be added to the maintainers list! :slightly_smiling_face:

**Comment by yajo** [comment]
That's very nice!

I've never used conda, so I don't think I'm a good maintainer there. Also maintaining copier itself is enough 😁

So it's very nice that someone else is doing that! 

If you want, however, there are ways to collaborate. For example: 

1. You can open a PR to advertise that package in our installation docs. 
2. Or open another one to automatically publish it to anaconda on new tags, just like with pypi. Just patch [the action that handles that process](https://github.com/copier-org/copier/blob/fa3515eff2cc6b69048ce5c66b5baef3efd5543f/.github/workflows/ci.yml#L86) and add anything you need. 

Thanks!

**Reply by kir0ul** [reply]
1. Here it is: https://github.com/copier-org/copier/pull/739
2. Apparently they have a different process on Conda-forge, there's a bot that watches for new releases and that should automatically open a PR on the Conda feedstock repo when it finds a new one. I'm not familiar with it so I guess we'll see at the next release if that works :sweat_smile:
